pkg/apiclient/apiclient.go

```diff
@@ -50,7 +50,7 @@ func NewClient(argoServer string, authSupplier func() string, clientConfig clien
 	})
 }
 
-func NewClientFromOpts(opts Opts) (context.Context, Client, error) {
+func NewClientFromOpts(ctx context.Context, opts Opts) (context.Context, Client, error) {
```

Comment:
this is a breaking change - instead - can we have `opts.context`?

Reply:
I moved the context to the `opts`.